web_src/css/modules/animations.css

```diff
@@ -35,6 +35,19 @@
   border-width: 2px;
 }
 
+form.is-loading::after {
+  max-height: 50%;
+  max-width: 50%;
+}
```

Comment:
I think these "max-" limits can be on `.is-loading`

Reply:
The result would be too small at least for inline math in markup. Could maybe be solved with additional `min-height` and `min-width`.

Reply:
Pushed a fix for this in https://github.com/go-gitea/gitea/pull/25380/commits/8c1639a6f288abc3ad32111bf1b2062d079a9878. `aspect-ratio` was not working because we set width and height. It works now that only height is set.